web/js/codeworld.js

```diff
@@ -158,6 +158,63 @@ function initCodeworld() {
     }
 }
 
+var cStream;
+var recorder;
+var chunks = [];
```

Comment:
I reset chunks to to an empty array after the recording is saved, but I'll clean this up and put it in a class instead of a global.